pywr/parameters/_parameters.pyx

```diff
@@ -1865,7 +1871,11 @@ def load_parameter(model, data, parameter_name=None):
         schema_cls = cls.Schema
         if schema_cls is not None:
             schema = schema_cls(context={'model': model, 'klass': cls})
-            parameter = schema.load(kwargs)
+            try:
+                parameter = schema.load(kwargs)
+            except ValueError as e:
+                raise ValueError(f"An error has occurred loading schema"
```

Comment:
I believe we should use the form:

```python
raise ValueError(f"An error has occurred ...") from e
```

This will cause the exception to state the `e` was the direct cause of the new `ValueError`. I think otherwise it will say while handling `e` another exception was generated.

Reply:
done.

Reply:
I can't see that this has been updated?